The new update isn't working on Hoobs after updating

I'm new here. I used the 0.3.3 for a long time, it worked flawlessly. But i was excited for this new release since i saw the awesome release notes from the pre-releases. I updated it today right after the release and all my accessories dissappeared. I tryed to unninstall the plugin completelly and try to install it again from the "Hoobs Plugin Search". Didn't work.

So I went to my log and saw this never-saw-before-message:

`"TuyaWebPlatform" which has not been published by any installed plugins.`

I went back to the 0.3.3 version (by restoring my saved hoobs backup made before updating the plugin) to make all work again.

Hope you can help me.

/hoobs/etc # npm install -g @milo525/homebridge-tuya-web
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@milo525%2fhomebridge-tuya-web - Not found
npm ERR! 404 
npm ERR! 404  '@milo525/homebridge-tuya-web@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,11 +1,11 @@
 import { API } from 'homebridge';
 
-import { PLATFORM_NAME } from './settings';
+import { PLATFORM_NAME, PLUGIN_NAME } from './settings';
 import { TuyaWebPlatform } from './platform';
 
 /**
  * This method registers the platform with Homebridge
  */
 export = (api: API) => {
-  api.registerPlatform(PLATFORM_NAME, TuyaWebPlatform);
+  api.registerPlatform(PLUGIN_NAME, PLATFORM_NAME, TuyaWebPlatform);
 }

